mypy/messages.py

```diff
@@ -897,6 +897,10 @@ def typeddict_item_name_not_found(self,
     def type_arguments_not_allowed(self, context: Context) -> None:
         self.fail('Parameterized generics cannot be used with class or instance checks', context)
 
+    def disallowed_any_type(self, typ: Type, context: Context) -> None:
+        self.fail('Expressions of type "Any" are disallowed '
```

Comment:
I'm not sure what you mean. 
Do you mean a situation when a type contains `Any` (e.g. `List[Any]`)?